js/models/chatmessage.js

```diff
@@ -72,6 +72,12 @@
 			}
 
 			return Backbone.Model.prototype.sync.call(this, method, model, options);
+		},
+
+		updateGuestName: function(data) {
+			if (this.get('actorType') === 'guests' && this.get('actorId') === data.sessionId && this.get('actorDisplayName') !== data.displayName) {
+				this.set('actorDisplayName', data.displayName);
```

Comment:
@danxuliu the models are updates, but the view of the chat messages needs to be rerendered. How can this be achieved?